futurenhs.data/FutureNHS.Data.FutureNHS/dbo/Tables/GroupUsersInRoles.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[GroupUsersInRoles]
```

Comment:
I think this table might need a rowversion too

Reply:
Yes it will do